modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -217,7 +227,7 @@ public void setUp() throws Exception {
 				false,
 				ListUtil.concat(
 					_createListTypeEntries(
-						"listTypeEntryKey", "List Type Entry Key ", 3),
+						"listTypeEntryKey", "List Type Entry Key ", 4),
```

Comment:
@thallescezar why was needed replace  3 with 4?

Reply:
As I added a new test scenario it was needed to increment this list of type entries.